packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -46,6 +47,9 @@ let MONGODB_VERSION = '';
 let MONGODB_USE_ENTERPRISE =
   (process.env.MONGODB_VERSION?.endsWith('-enterprise') && 'yes') ?? 'no';
 
+// should we test compass-web (true) or compass electron (false)?
+export const TEST_COMPASS_WEB = process.argv.includes('--test-compass-web');
```

Comment:
I don't know what to call this thing. We can easily find/replace it with a better name. Not sure if it is best to calculate it based off an env var or a cli parameter. We need to use this both in mocha tests and in helpers and having it as a const you can import seems like the cleanest of the options I can think of.